src/com/lftechnology/java/training/dipak/Test.java

```diff
@@ -0,0 +1,28 @@
+package com.lftechnology.java.training.dipak;
+
+import java.util.Scanner;
+
+public class Test {
+
+	public static void main(String[] args) {
+		Scanner scanner=new Scanner(System.in);
+		System.out.print("Enter your name with Salutation::");
+		String inputName=scanner.nextLine();
+		inputName=inputName.trim();
+		
+		String[] text=inputName.split(" ");
+		
+		System.out.println("Salutation::"+text[0]);
+		String name="";
+		
+		for(int i=1;i<text.length;i++){			
+			if(!(text[i].length()==0)){
+				name=name+text[i]+" ";
+			}					
+			
+		}
+		System.out.println("Your name::"+name);
+		scanner.close();

```

Comment:
do you think the scanner resource will be closed always ? There are two ways to make guarantee of these. 
1. Always use finally block to close the resources
2. User try-with-resource java 7 feature

http://www.mkyong.com/java/try-with-resources-example-in-jdk-7/
http://baptiste-wicht.com/posts/2010/08/java-7-try-with-resources-statement.html